src/main/java/org/testng/TestRunner.java

```diff
@@ -725,7 +724,7 @@ public int compareTo(IWorker<ITestNGMethod> other) {
       }
     });
 
-    runWorkers(workers, "" /* JUnit does not support parallel */, null);
+    runWorkers(workers);
```

Comment:
this method has been refactored since it's only called by junit test. Do you think we should rename the method as well?